pkgs/development/python-modules/libdecsync/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "libdecsync";
+  version = "1.7.1";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "IcdOOgkj3sfLuP+pkSEaHmqot+km3wLJNx+H/4ONz+g=";
+  };
+
+  meta = with lib; {
+    description = "libdecsync is a Python3 wrapper around libdecsync for synchronizing using DecSync";
```

Comment:
We probably need to include libdecsync somewhere, right?

Reply:
Yes, propogatedBuildInputs would be correct right?

Reply:
It seems it includes the .so and .dll files in the repo https://github.com/39aldo39/libdecsync-bindings-python3/tree/master/libdecsync/libs , I assume that complicates things?

Reply:
We want to compile those files and not use some random binary files from a git repo.

Reply:
Ah okay.